docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
+
+You must log on to the VEBA appliance as root. You can do this from the console. If you want SSH access, execute the following command:
+
+```
+systemctl start sshd 
```

Comment:
We default to allow root password logins if ssh is (temporarily) enabled, correct?

Reply:
@embano1 That's correct. SSH is disabled by default for security purposes. The suggested command above temporarily enables it. We could provide guidance to also disable it after troubleshooting has completed

Reply:
Added stop command